conviva/src/main/java/com/bitmovin/analytics/conviva/ConvivaAnalyticsIntegration.java

```diff
@@ -181,24 +135,23 @@ public void endSession() {
     public void updateContentMetadata(MetadataOverrides metadataOverrides) {
         this.contentMetadataBuilder.setOverrides(metadataOverrides);
         this.metadataOverrides = metadataOverrides;
-
-        if (!this.isSessionActive()) {
-            Log.i(TAG, "[ ConvivaAnalytics ] no active session; Don't propagate content metadata to conviva.");
-            return;
-        }
-
         this.createContentMetadata();
         this.updateSession();
     }
 
+    public void release() {
```

Comment:
Please add a comment to explain usage of this public API

Reply:
This is done in [this commit](https://github.com/bitmovin/bitmovin-player-android-analytics-conviva/pull/38/commits/80618197309f0e3441dd5254221fcfac33f25d84)